backoffice/src/components/DialogueBox.tsx

```diff
@@ -0,0 +1,54 @@
+import { FunctionComponent, useState } from "react";
+import Button from "@mui/material/Button";
+import Dialog from "@mui/material/Dialog";
+import DialogActions from "@mui/material/DialogActions";
+import DialogContent from "@mui/material/DialogContent";
+import DialogContentText from "@mui/material/DialogContentText";
+import DialogTitle from "@mui/material/DialogTitle";
+
+interface IDialogueBoxProps {
```

Comment:
I made this earlier on in the story but we're not using it now. Thought I would keep it in case Linda specifies she wants a dialogue box/if we need one for a future story